source/brailleDisplayDrivers/seika.py

```diff
@@ -1,25 +1,24 @@
 # -*- coding: UTF-8 -*-
-#brailleDisplayDrivers/seika.py
-#A part of NonVisual Desktop Access (NVDA)
-#This file is covered by the GNU General Public License.
-#See the file COPYING for more details.
-#Copyright (C) 2012 Ulf Beckmann <beckmann@flusoft.de>
-
-# Parts of this code are inherited from the baum braille driver
-# written by James Teh <jamie@jantrid.net>
-
+# brailleDisplayDrivers/seika.py
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2012/20 Ulf Beckmann <beckmann@flusoft.de>

```

Comment:
Please rewrite the copyright header in accordance with https://github.com/nvaccess/nvda/wiki/Copyright-headers